oqamaint/openqabot.py

```diff
@@ -408,7 +411,7 @@ def check_one_request(self, req):
             self.logger.error(traceback.format_exc())
             return None
 
-        return ret
+        return
```

Comment:
define ret=None at beginning of function and not needed explicit return on every if branch

Reply:
I know, but I like it better to have the returns more local. That's why I changed it to begin with - I just forgot about one branch